requirements.txt

```diff
@@ -5,5 +5,8 @@ Pillow
 ipywidgets>=0.6.0
 requests
 SimPEG
-em_examples
+git+https://github.com/geoscixyz/em_examples.git@tdem_groundedsource
```

Comment:
same question here. Just let me know if a new release needs to be uploaded (I am not sure the auto-deploy is up and running at the moment)